client/karma.js

```diff
@@ -120,14 +120,33 @@ function Karma (socket, iframe, opener, navigator, location, document) {
         }
         loadScript(0)
       }
-    // run in iframe
+      // run in iframe
     } else {
       iframe.src = policy.createURL(url)
     }
   }
 
+  /**
+   * Schedules te next execution after current context is cleared
+   * (or is directly started if context is currently not being cleared)
+   * @param execution {() => void}
+   * @see https://github.com/karma-runner/karma/issues/3424
+   */
+  this.scheduleExecution = function (execution) {
+    if (reloadingContext) {
+      // A context reload is in progress. Wait for it to complete before executing.
+      this.scheduledExecution = execution
```

Comment:
I think this one character member name difference is hard to read, could we use eg `executionScheduled`?